examples/Migration/RunExamples.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Migration;
+
+require __DIR__ . '/vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Migration\CampaignManagement\CreateCompleteCampaignAdwordsApiOnly;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\V1\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\V1\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\V1\GoogleAdsException;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\V1\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V1\Services\GoogleAdsRow;
+use Google\AdsApi\AdWords\AdWordsSessionBuilder;
+use Google\AdsApi\Common\OAuth2TokenBuilder as AdWordsApiOAuth2TokenBuilder;
+use Google\ApiCore\ApiException;
+
+/** This class creates the API client objects and runs the migration examples. */
+class RunExamples
+{
+    const DEVELOPER_TOKEN = 'INSERT_YOUR_DEVELOPER_TOKEN_HERE';
+    const CLIENT_ID = 'INSERT_YOUR_OAUTH2_CLIENT_ID_HERE';
+    const CLIENT_SECRET = 'INSERT_YOUR_OAUTH2_CLIENT_SECRET_HERE';
+    const REFRESH_TOKEN = 'INSERT_YOUR_OAUTH2_REFRESH_TOKEN_HERE';
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    const PAGE_SIZE = 1000;
+
+    // Login Customer ID is a new configuration setting required in Google Ads API. This
+    // is the customer ID of the authorized customer to use in the request, without
+    // hyphens(-). If your access to the customer account is through a manager account,
+    // this parameter is required and must be set to the customer ID of the manager account.
+    // See https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id
+    // to learn more about this setting.
+    const LOGIN_CUSTOMER_ID = 'INSERT_LOGIN_CUSTOMER_ID_HERE';
+
+    public static function main()
+    {
+        // Either pass the required parameters for this example on the command line, or insert them
+        // into the constants above.
+        $options = (new ArgumentParser())->parseCommandArguments([
```

Comment:
Can you just support all other parameters here too?

Reply:
We generally don't pass credentials as command line arguments, not even in the authentication examples, and I think it's because of security reasons, i.e. we don't want to have them in the shell history. Perhaps it's better to remove the customer id as a command line argument as well and have them all as constants at the top of the file.

WDYT?

Reply:
Yep. In that case, I agree we should remove customer ID from the parameter list too.
At the end, all users need to open this file and edit the source directly, so allowing passing parameters partially is not quite useful.

Reply:
Done.